editor/scene_create_dialog.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "scene_create_dialog.h"
 
+#include "core/config/project_settings.h"
```

Comment:
This `#include` is no longer needed as you are now using `Node::adjust_name_casing()` instead of `GLOBAL_GET()`.